src/app/DataModels/userMessageRequestModel.ts

```diff
@@ -0,0 +1,6 @@
+export interface UserMessageRequestModel {
```

Comment:
Usual naming convention is ISomething.

Reply:
From the code examples at https://angular.io/docs, it looks like the Hungarian notation is not used in Angular JS apps.